rita_common/src/tunnel_manager/shaping.rs

```diff
@@ -40,6 +41,11 @@ pub enum ShapingAdjustAction {
 }
 
 pub fn handle_shaping() {
+    if potential_payment_issues_detected() {
```

Comment:
this is the shaper, so it's not actually enforcing on the node, that's in mod.rs I think? (of tunnel manager) this is just adjusting the latency shaper up and down to keep latency smooth.